app/domain/errors.rb

```diff
@@ -0,0 +1,213 @@
+# frozen_string_literal: true
+
+require 'util/error_class'
+
+unless defined? Errors::Authentication::AuthenticatorNotFound
+  # this wrapper is here so these classes will not be loaded by Rails
```

Comment:
This wrapper prevents classes from being loaded by Rails

Reply:
this is actually a note for future developers but i'll change it :)